jenkinsapi/jenkins.py

```diff
@@ -268,6 +275,10 @@ def get_queue_url(self):
         url = "%s/%s" % (self.base_server_url(), 'queue')
         return url
 
+    @property
+    def get_use_baseurl(self):
```

Comment:
Why this getter is required? `use_baseurl` is not private variable and there is no use for this function besides unit tests, thus there is no need to have it.